VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx

```diff
@@ -35,7 +49,7 @@ const PrescriptionHistoryNoPrescriptions: FC = () => {
           <TextView pt={theme.paragraphSpacing.spacing20FontSize} paragraphSpacing={true} accessibilityLabel={a11yLabelVA(t('prescriptions.notFound.yourVA'))}>
             {t('prescriptions.notFound.yourVA')}
           </TextView>
-          <VABulletList listOfText={bullets} paragraphSpacing={true} />
+          <VABulletList listOfText={medicationNoIncludedList} paragraphSpacing={true} />
```

Comment:
This wasn't caused by the changes in this PR, but I noticed the spacing between the `Medications not included` text and the bullet point list is much greater than in the Figma mock

Reply:
Fixed the spacing to be compliant with WCAG